### PR TITLE
Homepage

### LINKED ISSUE #4: Homepage
- Create homepage
- DIsplay categories
- Create Footer

**Accepatance Criteria**: 
- Homepage with herosection
- CTA button to navigate to products

### PR DESCRIPTION
**Pull request by ttahm3d**
PR for Issue #4 

- Create homepage
- DIsplay categories
- Create Footer

Preview Image
localhost_3000_ (2)

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      <div className="main-container">
+        <div className={styles.footer__container}>
+          <div className="logo-container">
+            <img src={Logo} alt="logo" />
+          </div>
+          <div className="links-container">
+            <div className={styles.links__heading}>Quick Links</div>
+            <ul className={`${styles.list}`}>
+              <li className={styles.list__item}>
+                <Link to="/wishlist">Products</Link>
```
Comment (kjais1720) [inline src/components/Footer/Footer.jsx:17]: I think the route here is a typo,

File: src/index.css [hunk 2]
```diff
@@ -1,7 +1,25 @@
-@import url("https://chunksui.netlify.app/css/style.css");
+/* @import url("https://chunksui.netlify.app/css/style.css"); */
+@import url("https://unpkg.com/mockman-js@latest/dist/style.css");
+
+/* Overriding styles from component library */
```
Comment (kjais1720) [inline src/index.css:4]: I thin it'd be better if you imported the stylesheets in the HTML itself

**Review by kjais1720 — APPROVED** [review]
Everything looks good.
I really like the use of CSS modules and styled components

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (ttahm3d) [reply src/components/Footer/Footer.jsx:17]: I didnt have the route. I have changed it now.

File: src/index.css (hunk 2 above)
Comment (ttahm3d) [reply src/index.css:4]: For now I'm keeping it here

**Review by sadathshariff — APPROVED** [review]
The code Looks clean and like the way you have structured the files

**Pull request merged**